src/Console/Command/CompileCommand.php

```diff
@@ -180,9 +180,14 @@ private function buildPHP(string $spcBinaryPath, mixed $phpExtensions, mixed $ar
                 '--build-micro',
                 '--with-micro-fake-cli',
                 '--arch=' . $arch,
+                '--debug',
             ],
             cwd: $spcBinaryDir,
-            timeout: null,
+            env: [
+                'OPENSSL_LIBS' => '-l:libssl.a -l:libcrypto.a -ldl -lpthread',
+                'OPENSSL_CFLAGS' => sprintf('-I%s/source/openssl/include', $spcBinaryDir),
+            ],
```

Comment:
on my environment i add to force those value otherwise it would not found the ssl library

Reply:
since it use local lib, it should work in every case